runtime/compiler/codegen/J9RecognizedMethodsEnum.hpp

```diff
@@ -133,6 +133,9 @@
    java_lang_ref_Reference_refersTo,
    java_lang_ref_SoftReference_get,
    java_lang_Thread_currentThread,
+   java_lang_Thread_runWith,
+   java_lang_VirtualThread_runWith,
+   java_lang_ScopedValue_runWith,
```

Comment:
Within `ScopedValue`, there is an inner class `Carrier`; `runWith` is a method within `Carrier`: [jdk.lang.ScopedValue$Carrier.runWith](https://github.com/ibmruntimes/openj9-openjdk-jdk21/blob/78015256cbd4993cd51f522a6b4b5786a75dc00e/src/java.base/share/classes/java/lang/ScopedValue.java#L472).

Reply:
The convention for inner class methods for RecognizedMethodsEnum is to declare them as if they are part of the containing class. The actual string matching will get the right method.